projects/fusion-ui/components/chip-filter/v3/chip-filter.component.scss

```diff
@@ -0,0 +1,125 @@
+@import '../../../src/style/scss/v3/mixins/mixins';
+@import '../../../src/style/scss/v3/vars/colors';
+@import '../../../src/style/scss/v3/vars/fonts';
+
+
+%chip-font {
```

Comment:
this is not good. fonts shouldn't define in components. it should be a library setting. please use one of our fonts. if it not included then let design team know